### ISSUE TITLE
Intermittent error "Exiting with error as TwinGate is not connected"

### ISSUE THREAD

**Issue by jessebye** [Twingate/github-action#1]
We intermittently see this error in our workflows:

```
Starting Twingate service in headless mode
Twingate has been started
Waiting for status...
authenticating
Exiting with error as TwinGate is not connected
```

It seems like it should retry (with a timeout) until it gets the desired status, rather than immediately failing.

**Comment by rodrigocollavo** [comment]
We are experiencing the same problem and implementing our custom action with a retry mechanism, but it would be great to have an official solution and check if this more related to a timing/timeout issue to prevent a retry.

**Comment by ericbuehl** [comment]
It would appear this has been addressed in https://github.com/Twingate/github-action/pull/4.  Should this issue be closed? When can we expect a new release to be cut?

**Comment by ekampf** [comment]
Fixed in #4
